src/Spout/Writer/Style/BorderBuilder.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Box\Spout\Writer\Style;
+
+/**
+ * Class BorderBuilder
+ */
+class BorderBuilder
+{
+    /**
+     * @var Border
+     */
+    protected $border;
+
+    public function __construct()
+    {
+        $this->border = new Border();
+    }
+
+    /**
+     * @param string $style Border style @see BorderPart::allowedStyles
+     * @param string $color Border A RGB color code
+     * @param string $width Border width @see BorderPart::allowedWidths
+     * @return self
+     */
+    public function setBorderTop($style = Border::STYLE_SOLID, $color = Color::BLACK, $width = Border::WIDTH_THICK)
```

Comment:
Feels to me like a No-Win-Situation (After all we change this for the second time ;) ). From the top of my head ```color > width > style``` seems reasonable. I usually just use solid thin borders. I guess the color would change more likely.

Maybe we should change the default border width to thin or medium anyway (Thick is very "thick").

In order to prevent setting the styles again and again we could introduce static "default setters" for style, width and color. This would be something for the second pass on the border styling imo.

Or... convert to Python.